src/scripts.js

```diff
@@ -1,6 +1,20 @@
-import users from './data/users-data';
-import recipeData from  './data/recipe-data';
-import ingredientData from './data/ingredient-data';
+import { userGET, ingredientsGET, recipeGET } from './apiCalls'
+
+let users;
+let ingredientData;
+let recipeData;
+
+userGET().then(data => {
+  users = data.map(users => new User(users));
+})
+
+ingredientsGET().then(data => {
+  ingredientData = data;
+})
+
+recipeGET().then(data => {
+  recipeData = data;
+})
```

Comment:
We could try and refactor these similar `.then()` chains into a `Promise.all()` in the future